apps/console/src/components/shared/search/search.tsx

```diff
@@ -254,15 +285,19 @@ const renderOrgSearchResultField = ({ node, handleOrganizationSwitch, setQuery }
 }
 
 // renderSearchResultField is a generic function to render search result for any type of entity
-const renderSearchResultField = ({ node }: SearchGroupProps) => {
-  const { item } = searchStyles()
+interface SearchGroupProps {
+  node: any
+  searchType: string
+}
 
-  const nodeType = node.__typename.toLowerCase()
+const renderSearchResultField = ({ node, searchType }: SearchGroupProps) => {
+  const { item } = searchStyles()
+  const nodeType = searchType.toLowerCase()
 
   return (
     <CommandItem className={item()} key={node.id} onSelect={() => (window.location.href = `/${nodeType}/${node.id}`)}>
       <Link href={`/${nodeType}/${node.id}`}>
-        <div>{node.name}</div>
+        <div>{node.name || node.refCode || 'Unnamed'}</div>
```

Comment:
tasks have titles I believe so should have this I think
```suggestion
        <div>{node.name || node.refCode || node.title || 'Unnamed'}</div>
```